src/main/java/mods/railcraft/common/modules/ModuleAdvancedTanks.java

```diff
@@ -33,8 +28,8 @@
     public final static int CAPACITY_PER_BLOCK_OSMIUM = 2048 * FluidHelper.BUCKET_VOLUME;
     public final static int CAPACITY_PER_BLOCK_NEUTRONIUM = 3072 * FluidHelper.BUCKET_VOLUME;
 
-    public static final HashMap<String, IEnumMachine> cacheTankType = new HashMap<String, IEnumMachine>();
-    public static final HashMap<String, GenericMultiTankBase> cacheTankMaterial = new HashMap<String, GenericMultiTankBase>();
+    public static final HashMap<String, IEnumMachine> cacheTankType = new HashMap<>();
```

Comment:
Why remove the map types?

Reply:
It's implicit from the HashMap but I can put them back in

Reply:
>Implicit

Not really.
Your IDE will/should complain the typing is missing.
![image](https://user-images.githubusercontent.com/3060479/145108301-8867d236-e0cc-45b4-a261-90e4e3ed18ba.png)

Reply:
my IDE complains it's explicit hehehe 
![image](https://user-images.githubusercontent.com/533931/145108739-92b1d592-dc4b-4414-98b8-7cdf2edc36fa.png)
reverted tho